cpg-core/src/main/java/de/fraunhofer/aisec/cpg/helpers/Benchmark.kt

```diff
@@ -41,6 +41,19 @@ interface StatisticsHolder {
 
     fun addBenchmark(b: Benchmark)
 
+    fun getBenchmarkResult(): List<List<Any>> {
+        return listOf(
+            listOf("Translation config", config.toJson()),
+            listOf("Number of files translated", translatedFiles.size),
+            listOf(
+                "Translated file(s)",
+                translatedFiles.map { relativeOrAbsolute(Path.of(it), config.topLevel.toPath()) }
+            ),
+            *benchmarks
+                .map { listOf("${it.caller}: ${it.message}", "${it.duration} ms") }
+                .toTypedArray()
+        )
+    }
     /** Pretty-prints benchmark results for easy copying to GitHub issues. */
     fun printBenchmark() {
```

Comment:
Ideally, i would suspect that printBenchmark is just a printer and getBenchMarkResults does the actual retrieval of the different metrics. Can you rework it so that printBenchmarks calls getBenchmarkResult and prints the individual items in the list?

Reply:
I refactored it. My only concern was the special handling of the "Translation config". I moved that code part to `printMarkdown`. 
What was the reason for having this special format?

Reply:
The `toString` method is already printing the translation unit as a JSON. The idea was to have the TU as a json inside the markdown table.